pandas/_libs/tslibs/offsets.pyx

```diff
@@ -394,6 +394,15 @@ class _BaseOffset(object):
         out = '<%s' % n_str + className + plural + self._repr_attrs() + '>'
         return out
 
+    def _validate_n(self, n):
```

Comment:
can you add some tests to exercise this?

Reply:
Sure.  Got one test so far, will add a few more after addressing the current test failure (which is a bug that may need to be addressed as part of a Larger Fix).